### DISCUSSION TITLE
Feasibility of mouse/KB support

### DISCUSSION THREAD

**Discussion by amooose** [cathery/sys-con#70] [Feature requests]
I know it certainly wouldn't be easy, but would it be possible to implement sometime in the future?

**Comment by clancrate** [comment]
I would like to also see this implemented. Like we can customize the keys depending on our liking. For example, wasd for dpad. JIKOLP;[ for the other buttons. Depends on the user, as me myself use keyboard for fighting games. I know its weird but im more comfortable using a keyboard in some games.

**Comment by clancrate** [comment]
Even just the kb i would be fine already, is it possible?
